.github/workflows/rspec.yml

```
@@ -0,0 +1,28 @@
+name: RSpec
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+        ruby-version: ['2.7', '3.2']
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Ruby
```

Copy & pasting Gosu's requirements from Gosu's own GitHub Action...

```suggestion
    - uses: actions/checkout@v3
    
    - name: Install dependencies (Ubuntu)
        if: startsWith(matrix.platform, 'ubuntu-')
        run: sudo apt-get update && sudo apt-get install -y libsdl2-dev libgl1-mesa-dev libfontconfig1-dev xvfb

      - name: Install dependencies (macOS)
        if: startsWith(matrix.platform, 'macos-')
        run: brew install sdl2

    - name: Set up Ruby
```